src/Castle.Facilities.Logging/LoggingFacility.cs

```diff
@@ -103,34 +107,38 @@ public LoggingFacility(string customLoggerFactory, string configFile)
 		/// <param name="loggingApi"> The LoggerImplementation that should be used </param>
 		/// <param name="configFile"> The configuration file that should be used by the chosen LoggerImplementation </param>
 		/// <param name="customLoggerFactory"> The type name of the type of the custom logger factory. (only used when loggingApi is set to LoggerImplementation.Custom) </param>
+		[Obsolete("A logger factory implementation type should be provided via LogUsing<T>(), this will be removed in the future.")]
```

Comment:
There are currently 131 Obsolete attributes in the code base. Can't we just delete this altogether? Assuming we kick the version up to v5.